src/native/http_request_utils.h

```diff
@@ -36,6 +36,8 @@ struct aws_http_message *aws_http_request_new_from_java_http_request(
     jobjectArray jni_headers,
     jobject jni_body_stream);
 
+jobject aws_java_http_request_from_native(JNIEnv *env, struct aws_http_message *message);
+
```

Comment:
We'll also need a function that modifies an existing native aws_http_request to match the modifications made to the Java HtttpRequest? Can you add that in this PR please?